Guide.docc/CommonProblems.md

```diff
@@ -393,3 +393,312 @@ struct LivingIsland: Feedable {
 Here, a new type has been created that can satisfy the needed inheritance.
 Incorporating will be easiest if the conformance is only used internally by
 `Island`.
+
+## Crossing Isolation Boundaries
+
+Any value that needs to move from one isolation domain to another
+must be `Sendable`.
+Using non-`Sendable` values in contexts that require `Sendable` types is a very
```

Comment:
Nit: We shouldn't use the term "non-`Sendable` value". Instead, we should always talk about non-`Sendable` _types_ / types that don't conform to `Sendable` / values with non-`Sendable` type.

Reply:
arg we specifically talked about this but I make this mistake all the time.